MobiFlight/Config/MultiplexerDriver.cs

```diff
@@ -31,12 +31,8 @@ public bool isInitialized()
 
         public bool registerClient()
         {
-            bool res = false;
-            if (_initCounter > 0) {
-                _initCounter++;
-                res = true;
-            }
-            return res;
+            _initCounter++;
```

Comment:
This should be a `private` message because this is "inner" state that no one should worry about outside.

Reply:
Could you explain why should this be "inner"? (The variable obviously is.)
Un/Registering a new client for the driver is essential to keep count when - in the process of adding/deleting devices - the driver can be invalidated and its pins released; this is obviously an operation that has to be exposed.